sos/policies/package_managers/flatpak.py

```diff
@@ -15,16 +15,20 @@ class FlatpakPackageManager(PackageManager):
     """Package Manager for Flatpak distributions
     """
 
-    query_command = 'flatpak list --columns=name,version'
+    query_command = 'flatpak list --columns=name,version,branch'
     query_path_command = ''
     files_command = ''
     verify_command = ''
     verify_filter = ''
 
     def _parse_pkg_list(self, pkg_list):
-        for line in pkg_list.splitlines():
-            pkg = line.split()
-            name, version = pkg[0], pkg[1]
-            yield (name, version, None)
+        for line in pkg_list.splitlines()[1:]:
+            pkg = line.split("\t")
+            name, branch = pkg[0], pkg[2]
+            if not pkg[1]:
+                version = ''
+            else:
+                version = pkg[1]
+            yield (name, version, branch)
```

Comment:
Why not simple `yield (pkg[0], pkg[1], pkg[2])` ?  (maybe worth checking `len(pkg)>2`, just in case of truncated output (or that is over-paranoic..?))

Reply:
Checking the length of pkg and then yielding either pkg[1] or None seems better than just yielding pkg[1] regardless of it being there or not:

            if len(pkg) > 2:
                yield (pkg[0], pkg[1], pkg[2])
            else:
                yield (pkg[0], None, pkg[2])

Unless yielding pkg[1] when doesn't exist changes it to None on the yield, which then may simply the code a lot. But I saw in the other sources in package_managers that we explicitly set to None.

Reply:
Ah right, I am not sure now if the `split` returns `None` or empty string. But can we pass `None` as version? Does not we expect a string value later on?

Reply:
Later on I'm not sure, but if that's the case, we can fail back to "" instead of None. Let me see if I can test with packages with empty value and see what happens with None.

Reply:
OK, `split` returns empty string and not `None` (which is in fact obvious, it returns list of strings):

    >>> "name\tvers\tbranch".split("\t")
    ['name', 'vers', 'branch']
    >>> "name\t\tbranch".split("\t")
    ['name', '', 'branch']
    >>> 

And as far as I checked, we assume strings there.

Reply:
And I just checked what we get with yield (pkg[0], pkg[1], pkg[2]) without checking pkg lenght, from _generate_pkg_list, and it is an empty string. So just that line should be enough.